### PR TITLE
include web console plugin in setup script

### PR DESCRIPTION
**Pull request by sbryzak**

### CODE REVIEW

**Review by alexeykazakov — APPROVED** [review]
LGTM but let's make sure @MatousJobanek reviews it too

**Review by MatousJobanek — COMMENTED** [review]
please fix the issues that I mentioned in my comments, plus update your member-operator PR so it uses your fork and this branch of toolchain-cicd repo (to make sure that it works fine in e2e tests), to do that, replace this line: 
https://github.com/codeready-toolchain/member-operator/blob/8d9bed99fdff07dc8b1c9dfede5aa4dc1a7db8f2/make/run-cicd-script.mk#L1
with:
```
OWNER_AND_BRANCH_LOCATION=sbryzak/toolchain-cicd/add_consoleplugin_image
```

File: scripts/cd/olm-setup.sh [hunk 1]
```diff
@@ -220,10 +220,12 @@ generate_bundle() {
         CSV_SED_REPLACE+=";s|${EMBEDDED_REPO_REPLACEMENT}|${EMBEDDED_REPO_IMAGE}|g;"
     fi
     if [[ ${PRJ_NAME} == "member-operator" ]]; then
-        COMPONENT_IMAGE_URL=${COMPONENT_IMAGE:-quay.io/${QUAY_NAMESPACE_TO_PUSH}/member-operator-webhook:${GIT_COMMIT_ID}}
+        WEBHOOK_COMPONENT_IMAGE_URL=${COMPONENT_IMAGE:-quay.io/${QUAY_NAMESPACE_TO_PUSH}/member-operator-webhook:${GIT_COMMIT_ID}}
+        CONSOLEPLUGIN_COMPONENT_IMAGE_URL=${COMPONENT_IMAGE:-quay.io/${QUAY_NAMESPACE_TO_PUSH}/member-operator-console-plugin:${GIT_COMMIT_ID}}
```
[the remark this answers is not in the document]
Comment (sbryzak) [reply scripts/cd/olm-setup.sh]: done

File: scripts/cd/olm-setup.sh [hunk 2]
```diff
@@ -220,10 +220,12 @@ generate_bundle() {
...
         # digest format removed for now as it brought more pain than benefits
-        # COMPONENT_IMAGE_DIGEST_FORMAT=`get_digest_format ${COMPONENT_IMAGE_URL}`
```
[the remark this answers is not in the document]
Comment (sbryzak) [reply scripts/cd/olm-setup.sh]: fixed

File: scripts/cd/olm-setup.sh [hunk 3]
```diff
@@ -220,10 +220,12 @@ generate_bundle() {
...
-        CSV_SED_REPLACE+=";s|REPLACE_MEMBER_OPERATOR_WEBHOOK_IMAGE|${COMPONENT_IMAGE_URL}|g;"
+        # COMPONENT_IMAGE_DIGEST_FORMAT=`get_digest_format ${WEBHOOK_COMPONENT_IMAGE_URL}`
+        CSV_SED_REPLACE+=";s|REPLACE_MEMBER_OPERATOR_WEBHOOK_IMAGE|WEBHOOK_COMPONENT_IMAGE_URL|g;"
+        CSV_SED_REPLACE+=";s|REPLACE_MEMBER_OPERATOR_WEBCONSOLEPLUGIN_IMAGE|CONSOLEPLUGIN_COMPONENT_IMAGE_URL|g;"
```
[the remark this answers is not in the document]
Comment (sbryzak) [reply scripts/cd/olm-setup.sh]: fixed

**Review by MatousJobanek — APPROVED** [review]
This should work now - let's hope that we won't break anything.

**Pull request merged**